### PR TITLE
Add react-bootstrap-select@0.14.0 w/ npm autoupdate

### LINKED ISSUE #9062: [Request] Add react-bootstrap-select
**Library name:** react-bootstrap-select 
**Git repository url: https://github.com/tjwebb/react-bootstrap-select**
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #9062 
@kennynaoh Please help me reveiw it, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/tjwebb/react-bootstrap-select
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/react-bootstrap-select
 * GitHub / Bitbucket popularity (required):
   - Count of stars:46
   - Count of watchers:5
   - Count of forks:1525
 * NPM download stats (optional):
   - Downloads in the last day:38
   - Downloads in the last week:347
   - Downloads in the last month:1327

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

w/ npm auto-update config, close #9062, cc @tjwebb

### CODE REVIEW

**Comment by kennynaoh** [comment]
I think the main file is `js/bootstrap-select.js`.

**Comment by x09326** [comment]
@kennynaoh 
I added it from npm not from git repo.

**Comment by x09326** [comment]
@kennynaoh 
Why?
But only `Select.js` in the `dist` folder.

**Comment by x09326** [comment]
@kennynaoh 
I have added it and removed the `Select.js`.

File: ajax/libs/react-bootstrap-select/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "reactify": "^1.0.0",
+    "vinyl-buffer": "^1.0.0",
+    "vinyl-source-stream": "^1.0.0"
+  },
+  "npmName": "react-bootstrap-select",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "**/bootstrap-select.*"
```
Comment (kennynaoh) [inline ajax/libs/react-bootstrap-select/package.json]: How about `js/bootstrap-select*`?

Comment (x09326) [inline ajax/libs/react-bootstrap-select/package.json]: @kennynaoh 
I have modified it

**Comment by kennynaoh** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
No auto-update mentioned in PR title?

**Comment by x09326** [comment]
@PeterDaveHello 
Because the lib name is too long, if I mention auto-update then it will more than 50 chars.

**Comment by x09326** [comment]
@PeterDaveHello 
I meet a problem.
The master version is `0.14.0`.
But the latest commit time of `v1.0.0` is older than `0.14.0`.
And the author haven't reply my problem.
So I think the author will not maintain this lib.

**Comment by PeterDaveHello** [comment]
@x09326 I think we can just use the versions from npm, no v1.0.0 there.

**Comment by x09326** [comment]
@kennynaoh @PeterDaveHello 
I have added the `i18n` files.
Please help me review it, thanks.

**Comment by x09326** [comment]
@kennynaoh @PeterDaveHello 
The ci test has finished.
Please help me review it, thanks.

**Comment by x09326** [comment]
@pvnr0082t @kennynaoh 
I have added it.

**Comment by kennynaoh** [comment]
LGTM

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**